src/router/Router.jsx

```diff
@@ -17,6 +18,10 @@ const Router = () => {
       path: '/admin/main',
       element: <AdminMain />,
     },
+    {
+      path: '/admin/RequestList',
```

Comment:
```suggestion
      path: '/admin/requests',
```

url에는 대문자 사용을 지양해주세요